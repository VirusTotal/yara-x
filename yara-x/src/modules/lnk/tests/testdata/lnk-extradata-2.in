:100000004C0000000114020000000000C0000000CD
:1000100000000046EF4000002000000066BE89F4AA
:10002000F68ECA0166BE89F4F68ECA0100D4D9558F
:10003000668DCA0100AC0B0000000000010000004A
:100040000000000000000000000000005701140044
:100050001F50E04FD020EA3A6910A2D808002B3098
:10006000309D19002F433A5C0000000000000000A2
:1000700000000000000000000000008400310000CB
:10008000000000223C0978110050524F4752417E37
:100090003100006C0007000400EFBE6235515A24A5
:1000A0003C4CBB260000003D0000000000010000A9
:1000B000000000000000004200500072006F006766
:1000C00000720061006D002000460069006C006550
:1000D000007300000040007300680065006C006C55
:1000E00000330032002E0064006C006C002C002DE8
:1000F0000032003100370038003100000018004C99
:1001000000310000000000263CB88C1000506561F2
:100110005A69700000360007000400EFBE823B51B0
:10012000B3263CB88C2600000069B70000000082AE
:1001300000000000000000000000005000650061A9
:10014000005A0069007000000016005800320000DC
:10015000AC0B00243C408E20007065617A69702EE3
:1001600065786500003E0007000400EFBE263CB73E
:100170008C263CB78C26000000B6B900000000B207
:100180000000000000000000000000700065006139
:10019000007A00690070002E00650078006500009C
:1001A000001A000000590000001C000000010000BF
:1001B000001C000000350000000000000058000096
:1001C00000190000000300000050B37F4C10000035
:1001D00000766F6C756D655F6300433A5C50726FBB
:1001E0006772616D2046696C65735C5065615A6920
:1001F000705C7065617A69702E65786500002E000C
:100200005000650061005A00690070002000610024
:1002100072006300680069007600650072002C00BF
:100220002000610064006400200074006F00200062
:1002300065006E0063007200790070007400650054
:10024000640020002E007000650061002000610045
:1002500072006300680069007600650034002E00BB
:100260002E005C002E002E005C002E002E005C0094
:100270002E002E005C002E002E005C002E002E00B2
:100280005C002E002E005C00500072006F006700C2
:10029000720061006D002000460069006C0065007E
:1002A00073005C005000650061005A006900700036
:1002B0005C007000650061007A00690070002E002B
:1002C0006500780065000A002D006100640064008C
:1002D0003200630072007900700074003000430047
:1002E0003A005C00500072006F006700720061000D
:1002F0006D002000460069006C00650073005C0022
:100300005000650061005A00690070005C007200D6
:10031000650073005C00690063006F006E0073008D
:100320005C007000650061007A00690070005F0089
:100330006E00650077002E00690063006C001403F6
:100340000000070000A02550726F6772616D46695A
:100350006C6573255C5065615A69705C7265735C8D
:1003600069636F6E735C7065617A69705F6E6577E3
:100370002E69636C00000000000000000000000017
:10038000000000000000000000000000000000006D
:10039000000000000000000000000000000000005D
:1003A000000000000000000000000000000000004D
:1003B000000000000000000000000000000000003D
:1003C000000000000000000000000000000000002D
:1003D000000000000000000000000000000000001D
:1003E000000000000000000000000000000000000D
:1003F00000000000000000000000000000000000FD
:1004000000000000000000000000000000000000EC
:1004100000000000000000000000000000000000DC
:1004200000000000000000000000000000000000CC
:1004300000000000000000000000000000000000BC
:1004400000000000000000000000250050007200C5
:100450006F006700720061006D00460069006C006B
:100460006500730025005C005000650061005A00C3
:10047000690070005C007200650073005C00690038
:1004800063006F006E0073005C0070006500610027
:100490007A00690070005F006E00650077002E0032
:1004A000690063006C000000000000000000000014
:1004B000000000000000000000000000000000003C
:1004C000000000000000000000000000000000002C
:1004D000000000000000000000000000000000001C
:1004E000000000000000000000000000000000000C
:1004F00000000000000000000000000000000000FC
:1005000000000000000000000000000000000000EB
:1005100000000000000000000000000000000000DB
:1005200000000000000000000000000000000000CB
:1005300000000000000000000000000000000000BB
:1005400000000000000000000000000000000000AB
:10055000000000000000000000000000000000009B
:10056000000000000000000000000000000000008B
:10057000000000000000000000000000000000007B
:10058000000000000000000000000000000000006B
:10059000000000000000000000000000000000005B
:1005A000000000000000000000000000000000004B
:1005B000000000000000000000000000000000003B
:1005C000000000000000000000000000000000002B
:1005D000000000000000000000000000000000001B
:1005E000000000000000000000000000000000000B
:1005F00000000000000000000000000000000000FB
:1006000000000000000000000000000000000000EA
:1006100000000000000000000000000000000000DA
:1006200000000000000000000000000000000000CA
:1006300000000000000000000000000000000000BA
:1006400000000000000000000000000000000000AA
:10065000000010000000050000A02A000000B1000A
:1006600000001C0000000B0000A0EF405A7CFBA023
:10067000FC4B874AC0F2E0B9FA8EB100000060007E
:100680000000030000A0580000000000000070639C
:100690002D70726F70726965746172696F00CCD869
:1006A000920A4F36F644A3437CE76B0C19F52DCA2A
:1006B000A529E8FADE11A7060016E6D383ACCCD84C
:1006C000920A4F36F644A3437CE76B0C19F52DCA0A
:1006D000A529E8FADE11A7060016E6D383AC0000D0
:0206E000000018
:00000001FF
