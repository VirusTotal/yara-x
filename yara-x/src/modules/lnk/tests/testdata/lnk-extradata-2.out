is_lnk: true
creation_time: 1262799464
access_time: 1262799464
write_time: 1262627400
file_size: 764928
file_attribute_flags: 32
icon_index: 0
show_command: 1
hotkey_flags: 0
drive_type: 3
drive_serial_number: 1283437392
volume_label: "volume_c"
local_base_path: "C:\\Program Files\\PeaZip\\peazip.exe"
name: "PeaZip archiver, add to encrypted .pea archive"
relative_path: "..\\..\\..\\..\\..\\..\\..\\Program Files\\PeaZip\\peazip.exe"
arguments: "-add2crypt"
icon_location: "C:\\Program Files\\PeaZip\\res\\icons\\peazip_new.icl"
overlay_size: 0
link_target_id_list {
  size: 18
  data: "\037P\340O\320 \352:i\020\242\330\010\000+00\235"
}
link_target_id_list {
  size: 23
  data: "/C:\\\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
}
link_target_id_list {
  size: 130
  data: "1\000\000\000\000\000\"<\tx\021\000PROGRA~1\000\000l\000\007\000\004\000\357\276b5QZ$<L\273&\000\000\000=\000\000\000\000\000\001\000\000\000\000\000\000\000\000\000B\000P\000r\000o\000g\000r\000a\000m\000 \000F\000i\000l\000e\000s\000\000\000@\000s\000h\000e\000l\000l\0003\0002\000.\000d\000l\000l\000,\000-\0002\0001\0007\0008\0001\000\000\000\030\000"
}
link_target_id_list {
  size: 74
  data: "1\000\000\000\000\000&<\270\214\020\000PeaZip\000\0006\000\007\000\004\000\357\276\202;Q\263&<\270\214&\000\000\000i\267\000\000\000\000\202\000\000\000\000\000\000\000\000\000\000\000P\000e\000a\000Z\000i\000p\000\000\000\026\000"
}
link_target_id_list {
  size: 86
  data: "2\000\000\254\013\000$<@\216 \000peazip.exe\000\000>\000\007\000\004\000\357\276&<\267\214&<\267\214&\000\000\000\266\271\000\000\000\000\262\000\000\000\000\000\000\000\000\000\000\000p\000e\000a\000z\000i\000p\000.\000e\000x\000e\000\000\000\032\000"
}
